samples/bluetooth/per_adv/src/main.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <bluetooth/bluetooth.h>
+
+void main(void)
+{
+	struct bt_le_ext_adv_start_param adv_start_param = {
+		.timeout = 0,
+		.num_events = 0,
+	};
```

Comment:
Seeing as you added the other macros for extended adv, perhaps similar macros for the ext_adv_start_param could be added? I suppose most adv will just advertise without num_events or timeout so it would be generally useful. Can be added in another PR.

Reply:
I thought of adding, but as the Zephyr controller does not support connectable over extended advertising yet, I dont have/know any means to test it manually or in CI, hence not included in this PR. Yes, can be added in another PR.

Reply:
I'm sure I understand: What does the `bt_le_ext_adv_start_param` have to do with connectable advertising? I guess you could test such a macro with the sample added in this PR, couldn't you?

Reply:
Correction, my mistake, I was thinking of `BT_LE_ADV_CONN`et al. 

Yes, `bt_le_ext_adv_start_param` can have a define. Will give a try.